hack/test-cmd.sh

```diff
@@ -210,6 +210,20 @@ PATH="${KUBE_OUTPUT_HOSTBIN}":$PATH
 kube::log::status "Checking kubectl version"
 kubectl version
 
+# TODO: we need to note down the current default namespace and set back to this
+# namespace after the tests are done.
```

Comment:
Oh, I should had removed this TODO, because I noticed this script switches to an empty kubeconfig before running tests: https://github.com/kubernetes/kubernetes/blob/master/hack/test-cmd.sh#L139